lib/cisco_node_utils/command_reference_n7k.yaml

```diff
@@ -114,7 +114,19 @@ fabricpath:
     config_set: "%s fabricpath timers transition-delay %s"
     default_value: 10
 
-fp_topology:
+  ttl_multicast:
+    config_get: "show fabricpath ttl"
+    config_get_token: '/^Multicast TTL:\s+(\d+) hops/'
+    config_set: "%s fabricpath ttl multicast %s"
```

Comment:
The use of Key-value wildcards is preferred over Printf-style wildcards.

See: https://github.com/cisco/cisco-network-node-utils/blob/develop/docs/README-develop-best-practices.md#yaml8